CSS/style.css

```diff
@@ -0,0 +1,913 @@
+    background-color: #fff;
+    width: 90%;
+    height: 65px;
+    z-index: 5;
+    top: 40px;
+    padding-left: 10%;
+    padding-right: 5%;
+    margin: auto;
+    max-width: 1440px;
+  }
+
+  .logo img {
+    width: 95px;
+  }
+
+  .nav-top {
+    background: var(--text-color);
+    height: 40px;
+    width: 100%;
+    top: 0;
+    left: 0;
+    position: fixed;
+    margin: auto;
+    padding: 10px 40px 5px 10px;
+    padding-right: 40px;
+    z-index: 2;
+  }
+
+  .nav-top ul {
+    display: flex;
+    justify-content: end;
+    align-items: center;
+    gap: 30px;
+    color: #fff;
+    margin: auto;
+    max-width: 1440px;
+  }
+
+  .list-header {
+    display: flex;
+    gap: 35px;
+    align-items: center;
+  }
+
+  .list-header li a {
+    font-size: 20px;
+    font-weight: 400;
+  }
+
+  .list-header li:first-child a,
+  .list-header li:last-child a {
+    color: var(--main-color);
+  }
+
+  .list-header li:last-child {
+    color: var(--main-color);
+    border: 2px solid var(--main-color);
+    padding: 5px;
+  }
+
+  .hero {
+    background-image: url(/images/heroimg.jpg);
+    padding-bottom: 40px;
+    background-size: cover;
+    height: 100vh;
+    width: 90%;
+    margin: auto;
+  }
+
+  .hero-cont {
+    position: relative;
+    top: 194px;
+  }
+
+  .sect-two p {
+    text-align: center;
+    font-weight: 400;
+    font-size: 22px;
+  }
+
+  .sect-thre p {
+    text-align: center;
+    font-weight: 400;
+    font-size: 22px;
+  }
+
+  .cont-img h4,
+  .cont-img p {
+    z-index: 2;
+  }
+
+  .foot-text p {
+    font-size: 20px;
+    font-weight: 800;
+  }
+
+  .hero-cont p:first-child {
+    font-size: 18px;
+    color: var(--main-color);
+  }
+
+  .hero h1 {
+    color: var(--main-color);
+    margin: 10px 0;
+    font-size: 40px;
+  }
+
+  .hero-desp {
+    color: var(--text-color);
+    width: 70%;
+    font-size: 22px;
+    font-weight: 400;
+    margin: 40px 0;
+    border: 4px solid #fff;
+    background: #ede9e9;
+    padding: 15px;
+  }
+
+  .more-info h2 {
+    font-size: 40px;
+    font-weight: 800;
+    line-height: 1.8;
+    margin-top: 61px;
+    opacity: 0.5;
+  }
+
+  .program {
+    padding: 0 20px 40px 20px;
+    background-color: #272a31;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .program > h2 {
+    font-size: 36px;
+    text-align: center;
+    color: #fff;
+    padding: 40px 10px 22px 10px;
+    width: 300px;
+    margin: 0 auto 40px auto;
+    position: relative;
+  }
+
+  .program > h2::before {
+    content: '';
+    width: 50px;
+    height: 2px;
+    position: absolute;
+    background-color: var(--main-color);
+    left: 120px;
+    bottom: 0;
+  }
+
+  .cards {
+    display: flex;
+    gap: 3px;
+    padding-bottom: 20px;
+    height: 60vh;
+  }
+
+  .card-prog {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    gap: 20px;
+    padding: 20px 10px 40px 10px;
+    background-color: rgb(86, 86, 86);
+    min-height: 400px;
```

Comment:
- [ ] I think your cards are too big compared to the design. How about you reduce the `height` a bit? 
- [ ] I think you can use the `height` property instead of the `min-height` and give it something like `300px`.